packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -534,6 +546,31 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
     _manager?._markPropertiesChanged(this);
   }
 
+  /// If false, tells the focus traversal policy to skip over for all of this
+  /// node's descendants for purposes of the traversal algorithm.
+  ///
+  /// Defaults to true. Does not affect the focus traversal of this node: for
+  /// that, use [skipTraversal].
+  ///
+  /// See also:
+  ///
+  ///  * [ExcludeFocusTraversal], a widget that uses this property to conditionally
```

Comment:
You might also add a reference to `[ExcludeFocus]` and `[descendantsAreFocusable]`, in case the reader is really looking for those.

Reply:
Done.

Reply:
Although, I feel adding just `[descendantsAreFocusable]` should suffice, as that references `[ExcludeFocus]` anyway.